### ISSUE TITLE
Menu

### ISSUE THREAD

**Issue by kilchenmann** [dhlab-basel/Salsah-admin#3]
Create a user and / or app menu. It should look as follow:

![screen shot 2017-02-15 at 08 56 12](https://user-images.githubusercontent.com/4253580/44907254-95919800-ad17-11e8-85d4-221832ef97c1.png)

**Comment by kilchenmann** [comment]
What are the most important tools in the top right corner of the header toolbar?

Get inspired by the following screenshots:

![screen shot 2018-04-11 at 14 53 23](https://user-images.githubusercontent.com/4253580/44907781-70058e00-ad19-11e8-95f3-9ca85ed9ef85.png)

![screen shot 2018-04-11 at 14 53 50](https://user-images.githubusercontent.com/4253580/44907782-70058e00-ad19-11e8-9eb9-c2495801e4f4.png)

![screen shot 2018-04-11 at 14 54 34](https://user-images.githubusercontent.com/4253580/44907783-70058e00-ad19-11e8-8989-45105bbaa29e.png)

**Comment by flavens** [comment]
usually, the tools on the right corner are all the functionality about login, account/profile, settings, help/info, notifications, search

**Comment by kilchenmann** [comment]
Yes thatˋs right. But which one do we need ins salsah-admin and how should we arrange them? I like the idea of a user menu as we have now in Salsah. Bit then, do we have an add-menu? Where is the language switch?

**Comment by kilchenmann** [comment]
I suggest to have the language switch inside of the user menu. And then we have only an add (+) and a user menu
